### DISCUSSION TITLE
Undefined at v3.0.0-beta-2044

### DISCUSSION THREAD

**Discussion by viceice** [sebastienros/jint#1369] [Q&A] [answered]
Before `v3.0.0-beta-2044` i used `Undefined.Instance`. what's the new way?

should I return `new JsUndefined()`? or is there a global instance anywhere else

**Comment by Genteure** [comment, marked as answer]
You can use `JsValue.Undefined`.

**Reply by lahma** [reply]
Indeed, I've updated [the release notes](https://github.com/sebastienros/jint/releases/tag/v3.0.0-beta-2044) as it was lacking this change.

**Reply by viceice** [reply]
thanks 🙏
